src/NuGet.Core/NuGet.Commands/CommandArgs/VerbArgs.tt

```diff
@@ -9,7 +9,7 @@
 <#
     string InitCaps(string input)
     {
-        string output = input.Substring(0,1).ToUpper() + input.Substring(1);
+        string output = input.Substring(0, 1).ToUpper() + input.Substring(1);
```

Comment:
string.format would be better than +

Reply:
Not from a performance perspective. The book "Writing High-Performance .NET Code" by Ben Watson talked about this. string.format has to parse the format string, which is slower than string concatenation.

Reply:
Donnie and I chatted offline. Since this is the codegen part of the code, we decided to not worry about this feedback.